pkg/controller/cyclenoderequest/transitioner/transitions.go

```diff
@@ -34,10 +40,10 @@ func (t *CycleNodeRequestTransitioner) transitionUndefined() (reconcile.Result,
 		return t.transitionToHealing(fmt.Errorf("selector cannot be empty"))
 	}
 
-	if t.rm.Notifier != nil {
-		if err := t.rm.Notifier.CyclingStarted(t.cycleNodeRequest); err != nil {
-			t.rm.Logger.Error(err, "Unable to post message to messaging provider", "phase", t.cycleNodeRequest.Status.Phase)
-		}
+	// Protect against failure case where cyclops checks for leftover CycleNodeStatus objects using the CycleNodeRequest name in the label selector
+	// Label values must be no more than 63 characters long
+	if len(t.cycleNodeRequest.Name) > 63 {
```

Comment:
should use the kubernetes `validation` package. "k8s.io/apimachinery/pkg/util/validation"  validation.IsDNS1035Label probably for object names